spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/ReLU.scala

```diff
@@ -31,6 +32,9 @@ import scala.reflect.ClassTag
 @SerialVersionUID(1208478077576570643L)
 class ReLU[T: ClassTag](ip: Boolean = false)(
   implicit ev: TensorNumeric[T]) extends Threshold[T](0, 0, ip) {
+
+  override def computeOutputShape(inputShape: Activity): Activity = inputShape
```

Comment:
I think there's a lot of layers of which outshape == inputshape, we may need to create a trait for it?

Reply:
We can put this as default implementation in `AbstractModule`, but the bad thing is we need to review all of the existing layers to adapt to this which is not we want for now.
We can refactor this once all of layers support `computeOutputShpae`

Reply:
This is not the default implementation. There're some common patterns but no one should be the default. Of course, we need to change all existing layers.

Reply:
You should not put this as default impl, this may cause problem when people add new layers. We should use throw exception as default or change it to an abstract method.